packages-exp/app-exp/use_public_typings.js

```diff
@@ -16,16 +17,23 @@
  */
 
 const { writeFileSync } = require('fs');
+const { argv } = require('yargs');
```

Comment:
Since the script takes an argument now, rename it to `use_typings.js`?

Reply:
Done.